image/image_linux.go

```diff
@@ -264,6 +267,22 @@ func makeLabel(now time.Time) string {
 	return now.UTC().Format("20060102")
 }
 
+func saveAssertion(db *asserts.Database, as asserts.Assertion, model *asserts.Model) error {
+	tsto, err := newToolingStoreFromModel(model, "")
+	if err != nil {
+		return err
+	}
+	newFetcher := func(save func(asserts.Assertion) error) asserts.Fetcher {
+		return tsto.AssertionFetcher(db, save)
+	}
+
+	f := seedwriter.MakeRefAssertsFetcher(newFetcher)
```

Comment:
to do the right thing with respect to account-key we will need to get hold of f to look at f.Refs() I think